src/Microsoft.Extensions.ODataClient.Abstractions/ClientCreatedArgs.cs

```diff
@@ -0,0 +1,38 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ClientCreatedArgs.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.Extensions.ODataClient

```

Comment:
Should these be in the .Abstractions namespace?

Reply:
No, that's the namespace conventions all other Microsoft.Extensions.*.Absractions use, and I would agree on that because that make user experience simpler also.